docs/Documentation/UserGuideV0.7.0/4-Deployment and Management.md

```diff
@@ -512,6 +512,131 @@ Currently, IoTDB provides users to use Java's JConsole tool to monitor system st
 
 After starting JConsole tool and connecting to IoTDB server, you will have a basic look at IoTDB system status(CPU Occupation, in-memory information, etc.). See [official documentation](https://docs.oracle.com/javase/7/docs/technotes/guides/management/jconsole.html) for more informations.
 
+#### JMX MBean Monitoring
```

Comment:
If you add new chapters to document, please update the catalog above.
You can use VSCode and markdown plugin to auto-update catalog

Reply:
fixed